src/System.Xml.ReaderWriter/ref/System.Xml.ReaderWriter.cs

```diff
@@ -1365,28 +1397,39 @@ public sealed partial class XmlAtomicValue : System.Xml.XPath.XPathItem, System.
         public override string ToString() { throw null; }
         public override object ValueAs(System.Type type, System.Xml.IXmlNamespaceResolver nsResolver) { throw null; }
     }
+    [System.Xml.Serialization.XmlRootAttribute("schema", Namespace="http://www.w3.org/2001/XMLSchema")]
     public partial class XmlSchema : System.Xml.Schema.XmlSchemaObject
     {
         public const string InstanceNamespace = "http://www.w3.org/2001/XMLSchema-instance";
         public const string Namespace = "http://www.w3.org/2001/XMLSchema";
         public XmlSchema() { }
         [System.ComponentModel.DefaultValueAttribute(System.Xml.Schema.XmlSchemaForm.None)]
+        [System.Xml.Serialization.XmlAttributeAttribute("attributeFormDefault")]
```

Comment:
Are these attributes required in the ref? If so, can we fix this in the tool? cc @ericstj

Reply:
https://github.com/dotnet/corefx/blob/master/eng/DefaultGenApiDocIds.txt